data/battle_scripts_1.s

```diff
@@ -9151,16 +9151,23 @@ BattleScript_MirrorHerbCopyStatChange::
 	printstring STRINGID_MIRRORHERBCOPIED
 	waitmessage B_WAIT_TIME_LONG
 	removeitem BS_SCRIPTING
-	call BattleScript_TotemVar_Ret
-	copybyte gBattlerAttacker, sSAVED_BATTLER	@ restore the original attacker just to be safe
+BattleScript_MirrorHerbStartCopyStats:
```

Comment:
As discussed on Discord, this actually should function the same way as Opportunist.